scheduler/src/cook/rest/api.clj

```diff
@@ -966,6 +981,8 @@
     (s/validate Job munged)
     (when (and (:gpus munged) (not gpu-enabled?))
       (throw (ex-info (str "GPU support is not enabled") {:gpus gpus})))
+    ; Note that we are passing munged here because the function expected stringified env
+    (validate-job-gpu-model pool-name munged)
```

Comment:
Do we want to inline this to be consistent with the other checks? Once you extract the environment and the COOK_GPU_MODEL, this is only two lines to implement the check.

Reply:
Putting in its own function makes it significantly easier to unit test.